stl/inc/algorithm

```diff
@@ -1957,28 +2082,22 @@ namespace ranges {
         template <class _It1, class _It2, class _Pr, class _Pj1, class _Pj2>
         _NODISCARD static constexpr subrange<_It1> _Bidi_common_ranges(
             _It1 _First1, _It1 _Last1, _It2 _First2, _It2 _Last2, _Pr _Pred, _Pj1 _Proj1, _Pj2 _Proj2) {
-            auto _UFirst1       = _Get_unwrapped(_First1);
-            const auto _ULast1  = _Get_unwrapped(_Last1);
-            const auto _UFirst2 = _Get_unwrapped(_First2);
-            const auto _ULast2  = _Get_unwrapped(_Last2);
-            for (auto _UCandidate = _ULast1;; --_UCandidate) { // try a match at _UCandidate
-                auto _UNext1 = _UCandidate;
-                auto _UNext2 = _ULast2;
-                for (;;) { // test if [_UFirst2, _ULast2) is a suffix of [_UFirst1, _UCandidate)
-                    if (_UFirst2 == _UNext2) { // match found
-                        _Seek_wrapped(_First1, _UNext1);
-                        _Seek_wrapped(_Last1, _UCandidate);
+            for (auto _Candidate = _Last1;; --_Candidate) { // try a match at _Candidate
+                auto _Next1 = _Candidate;
+                auto _Next2 = _Last2;
+                for (;;) { // test if [_First2, _Last2) is a suffix of [_First1, _Candidate)
+                    if (_First2 == _Next2) { // match found
                         return {_First1, _Last1};

```

Comment:
Unlike the random-access implementation, this doesn't appear to move iterators when returning.